Makefile

```diff
@@ -13,20 +13,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# The goals assume that all Python package dependencies are
```

Comment:
Nit. I believe it is called Makefile `targets` and not `goals`.

```suggestion
# The targets assume that all Python package dependencies are
```

Reply:
Yes, "targets" would be a better fit in this case. Goals are what you as a user want to build (foo and bar in the case of `make foo bar`) and targets are the entries in the makefile that can fulfill goals.